src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/SemanticTokenTestBase.cs

```diff
@@ -129,7 +145,7 @@ private static void GenerateSemanticEditBaseline(SemanticTokensFullOrDelta edits
                     {
                         builder.Append(i).Append(' ');
                     }
-                    builder.AppendLine("] ");
+                    builder.AppendLine("]");
```

Comment:
Small housekeeping thing to keep trailing spaces off, the newline will delimit this.